spec/cart_spec.rb

```diff
@@ -11,13 +11,13 @@
 
     describe "#set_discounted" do
       it "sets new value for discounted" do
-        cart.set_discounted 5
-        cart.discounted.should be 5
+        cart.set_discounted 'some code'
+        cart.discounted.should be_eql 'some code'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.